gef.py

```diff
@@ -11116,8 +11160,8 @@ def file(self) -> pathlib.Path:
             filename = progspace.filename
             if not filename:
                 raise RuntimeError("No session started")
-            start_idx = len("target:") if filename.startswith("target:") else 0
-            self._file = pathlib.Path(filename[start_idx:])
+            start_idx = len("target:") if progspace.filename.startswith("target:") else 0
```

Comment:
Why not use `filename`? You assign it a few lines up, probably for brevity, but don't use it?

Reply:
Fixed